src/ui/Errors/Loading/Loading.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Box from 'ui/Box/Box';
+import Text from 'ui/Text/Text';
+import errorGear from './error-gear.svg';
+
+const ErrorGearImage = styled('img')`
+  height: 78px;
+  margin-bottom: 32px;
+  margin-left: 12px;
+  margin-top: 28px;
+  width: 84px;
+`;
+
+const MainText = styled.span`
+  color: ${props => props.theme.colors.solitude.main};
+  font-weight: 600;
+  line-height: 24px;
+  font-size: 16px;
+`;
+
+const OffText = styled.span`
+  color: ${props => props.theme.colors.solitude.main};
+  font-weight: 400;
+  line-height: 22px;
+  font-size: 15px;
+`;
+
+const ListLoadingErrorState = ({ children, mainText }) => (
+  <Box column hAlignContent="center">
```

Comment:
We're refraining from using the `Box` component as it introduces all sorts of console warnings now. I'd use a `styled.div`